mypy.ini

```diff
@@ -1,5 +1,8 @@
 [mypy]
 
+[mypy-Bio.*]
+ignore_missing_imports = True
+
 [mypy-ete3.*]
```

Comment:
Is this still necessary if you have removed it? Or is it just here in case we want to use it for future tests?

Reply:
Biopython is being used in [`filecmp.py`](https://github.com/Ensembl/ensembl-compara/blob/b9049ed9d3068d971428c3fe8474a468d936ff4a/src/python/lib/ensembl/compara/filesys/filecmp.py). But anyway, yes, we expect to be using this library in the future, so it can be added now.